### PR TITLE
Start of dev for PNW PLSE 2024

### PR DESCRIPTION
**Pull request by jyoo980**

### CODE REVIEW

**Comment by jyoo980** [comment]
Here are some screenshots 
image
image

File: img/pnwplse-logo-designs.key
Comment (ztatlock) [inline img/pnwplse-logo-designs.key:1]: Let's leave this in the repo. I believe it is the definitive copy of the logo design.

File: img/pnwplse.png
Comment (ztatlock) [inline img/pnwplse.png:1]: Similarly, I think we probably want to leave this in the repo so we have the logo handy.

File: style.css
Comment (ztatlock) [inline style.css:1]: What's the provenance on this CSS? PLSE page?

File: index.html
Comment (ztatlock) [inline index.html:1]: No `<html>` / `<head>` / `<body>` ?! Am I living in the future??

**Review by ztatlock — CHANGES_REQUESTED** [review]
@jyoo980 left some comments. I think only big thing is keeping the old logo files. Many thanks for pushing this through!!

File: style.css
Comment (jyoo980) [reply style.css:1]: Yep, it's ripped from the PLSE page but with minor style modifications

**Review by ztatlock — APPROVED** [review]
Looks great! Many thanks again for pushing on this!!

**Comment by jyoo980** [comment]
Will merge once Kevin gives the OK on the Google Form

**Pull request merged**